src/test/scala/info/bethard/timenorm/formal/TypesTest.scala

```diff
@@ -1037,10 +1050,11 @@ class TypesTest extends FunSuite with TypesSuite {
     assert(NextRI(SimpleInterval.of(1989, 11, 12), nov13) === SimpleInterval.of(1989, 11, 13))
   }
 
-  test("NYT19980206.0460 (2979,3004) first nine months of 1997") {
```

Comment:
There seems to be a merge error here. The NYT19980206.0460 test should not be deleted. Also, the 69@e@APW19980306.1001@isma5916 test that you've added seems wrong. 1998-03-06 was a Friday, so if you want to get back the same day, you need to do a `LastFromEndRI`, not a `LastRI`.